src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php

```diff
@@ -53,6 +53,7 @@ protected function configure()
                 new InputOption('docroot', 'd', InputOption::VALUE_REQUIRED, 'Document root'),
                 new InputOption('router', 'r', InputOption::VALUE_REQUIRED, 'Path to custom router script'),
                 new InputOption('pidfile', null, InputOption::VALUE_REQUIRED, 'PID file'),
+                new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (txt or json)', 'txt'),
```

Comment:
Since the only output being formatted to json is the hostname and port. This should probably be specified more or output all format as json.